config/broker/300-rabbitmqbrokerconfig.yaml

```diff
@@ -54,8 +54,8 @@ spec:
                   connectionSecret:
                     description: Secret contains the http management uri for the RabbitMQ
                       cluster. The Secret must contain the key `uri`, `username` and
-                      `password` or operator will error. Have to set either name or
-                      connectionSecret, but not both.
+                      `password` or operator will error, it may contain also `port` (default 5672).
```

Comment:
The Secret must contain the key `uri`, `username` and `password`. It may optionally contain `port` or will be defaulted to 5672